BaikalStandalone/Application/cl_render.h

```diff
@@ -115,6 +115,7 @@ namespace Baikal
         std::promise<int> m_promise;
         bool m_shape_id_requested = false;
         OutputData m_shape_id_data;
+        OutputData m_dummy_output_data;
```

Comment:
What this for?

Reply:
Opacity and visibility outputs require color output to be set, but we don't have another one viewport to render color there, so we use a "dummy" output